health/README.md

```diff
@@ -0,0 +1,29 @@
+# Health Package
+
+Package contains tools that allow to add REST endpoint for handling `health` and `readiness` requests.
+Package also provides implementation for most typical checks that may be done to figure our if service is healthy/ready.
+
+## REST Handler
+
+A handler to serve checks using `/health` and `/ready` endpoints can be added to your service in a following way: 
+
+```go
+	healthChecker := health.NewChecksHandler("/health", "/ready"),
+	)
+	healthChecker.AddReadiness("Readiness check name", readinessCheck)
+	healthChecker.AddLiveness("Liveness check name", livenessCheck)
+
+	s, err := server.NewServer(
+		// register our health checks
+		server.WithHealthChecks(healthChecker),
+	)
+```
+
+In a sample above `readinessCheck` and `livenessCheck` are functions that perform a single check.
+It is possible to add multiple checks independently and handler will call them in sequence.
+
+## Checks
+
+Packange provides several predefined checks:
+ - `DNSProbeCheck(host string, timeout time.Duration)` returns a Check that determines whether a service with specified dns name is reachable or not using net.Resolver's LookupHost method.
+ - `HTTPGetCheck(url string, timeout time.Duration)` returns a Check that performs an HTTP GET request to the specified URL. It fails if timeout was exceeded or non-200-OK status code returned.
```

Comment:
Might help to change Check to `Check` to denote that it's a `struct` inside the package.